doc/classes/UDPServer.xml

```diff
@@ -21,20 +22,57 @@
 		    server.poll() # Important!
 		    if server.is_connection_available():
 		        var peer : PacketPeerUDP = server.take_connection()
-		        var pkt = peer.get_packet()
+		        var packet = peer.get_packet()
 		        print("Accepted peer: %s:%s" % [peer.get_packet_ip(), peer.get_packet_port()])
-		        print("Received data: %s" % [pkt.get_string_from_utf8()])
+		        print("Received data: %s" % [packet.get_string_from_utf8()])
 		        # Reply so it knows we received the message.
-		        peer.put_packet(pkt)
+		        peer.put_packet(packet)
 		        # Keep a reference so we can keep contacting the remote peer.
 		        peers.append(peer)
 
 		    for i in range(0, peers.size()):
 		        pass # Do something with the connected peers.
+		[/gdscript]
+		[csharp]
+		using Godot;
+		using System;
+		using System.Collections.Generic;
 
-		[/codeblock]
-		[codeblock]
-		# client.gd
+		public class Server : Node
+		{
+		    public UDPServer Server = new UDPServer();
+		    public List&lt;PacketPeerUDP&gt; Peers = new List&lt;PacketPeerUDP&gt;();
+
+		    public override void _Ready()
+		    {
+		        Server.Listen(4242);
+		    }
+
+		    public override void _Process(float delta)
+		    {
+		        Server.Poll(); // Important!
+		        if (Server.IsConnectionAvailable())
+		        {
+		            PacketPeerUDP peer = Server.TakeConnection();
+		            byte[] packet = peer.GetPacket();
+		            GD.Print($"Accepted Peer: {peer.GetPacketIp()}:{peer.GetPacketPort()}");
+		            GD.Print($"Received Data: {packet.GetStringFromUTF8}");
```

Comment:
I think this is supposed to be a method call? If so it should have `()`.

Reply:
done